bool all
---
ball_picker/Detections detections
